wsl-builder/common/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ubuntu/WSL-DsitroLauncher/wsl-builder/common
```

Comment:
This is more than a FYI than something that needs an immediate action:
In Go, we generally avoid package and modules that are named `common` or `utils`. This was the name of the directory ofc with the symlink hack to avoid creating separate modules, but now that it’s a proper module, I’m a little bit more annoyed (yes, that’s a nitpick) :)

However, I don’t find a better naming. Something like `release` will make us then rename any structs/methods to avoid stuttering as they will be `release.` For instance `ReleaseInfo` -> `Info` (as `Release.Info`).

But there is GetPath will end up here while not being too close to the "Release" package.

So, let’s keep it that way, (told you it’s a FYI in general in Go), apart if you can come up with a better name :)

Reply:
BTW I didn't make everything public. Some functions were only used inside the package, so I didn't touch them. I think we should keep it that way until we find the need for them outside of the package. It is not meant to be used publicly outside of this project, so I think we will notice when that need arise. Is that OK for you?